content/millennium/r4/base/entities/location.md

```diff
@@ -60,24 +60,26 @@ Search for Locations that meet supplied query parameters:
- [`_count`]           | no                      | [`number`]   | The maximum number of results to return. Defaults to `100`.
- `address`            | no                      | [`string`]   | A (part of the) address of the location.
- `address-city`       | no                      | [`string`]   | A city specified in an address
- `address-state`      | no                      | [`string`]   | A state specified in an address
- `address-postalcode` | no                      | [`string`]   | A postal code specified in an address
- `name`               | no                      | [`string`]   | A portion of the location's name or alias
- `organization`       | no                      | [`reference`]| Searches for locations that are managed by the provided organization
+ `_id`                | Conditionally           | [`token`]    | The logical resource ID associated with the resource. This parameter is required if the `-physicalType` parameter is not used.
+ `-physicalType`      | Conditionally           | [`token`]    | The location’s physical type. This parameter is required if the `_id` parameter is not used. Example: `http://terminology.hl7.org/CodeSystem/location-physical-type|ro`
+ `identifier`         | No                      | [`token`]    | The location’s identifier. Example: `653385|FSI^~BUILD^~NU` 
+ [`_count`]           | No                      | [`number`]   | The maximum number of results to return. Default: `100`
+ `address`            | No                      | [`string`]   | Part of the location's address. Example: `123%20Main`
+ `address-city`       | No                      | [`string`]   | The city specified in the address. Example: `Kansas`
+ `address-state`      | No                      | [`string`]   | The state specified in the address. Example: `MO`
+ `address-postalcode` | No                      | [`string`]   | The postal code specified in the address. Example: `64111`
+ `name`               | No                      | [`string`]   | The portion of the location's name or alias. Example: `Main`
+ `organization`       | No                      | [`reference`]| The organization that manages the location. Example: `675844`
 
 
  Notes:
 
-- The `-physicalType` parameter
-  - Searching by Millennium proprietary codes is not supported.
-- The `-address-city` parameter
-  - needs address-state or address-postalcode while searching with address-city
+- When searching with the `-physicalType` parameter:
+  - Searching by Cerner Millennium proprietary codes is not supported.
+- When searching with the `-address-city` parameter:
+  - Must include any of the following parameters: `address-state` or `address-postalcode`.
+- When searching with the `name` and `organization` parameters:
+  - Must include any of the following parameters: `-physicalType`, `identifier`, `address`, `address-state`, `address-city`, or `address-postalcode`.
```

Comment:
do we need to mention `-physicalType` here since we've already stated that it's required?